index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Trybe Warts</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="header">
+    <img src="images/trybewarts-header-logo.svg" alt="trybe warts logo" class="trybewarts-header-logo">
+    <h1 id="trybewarts-header-title">Trybewarts</h1>
+    <form class="trybewarts-login" action="" method="GET">
+      <input type="text" required  placeholder="Login" id="input-login">
```

Comment:
O que acham de remover esse espaço duplo aqui? :P
```suggestion
      <input type="text" required placeholder="Login" id="input-login">
```

Reply:
Tem outros lugares nesse arquivo HTML que tem esse mesmo comportamento. O que acham de fazer uma busca com o termo de dois espaços? Ficaria algo do tipo:  `  `